### DISCUSSION TITLE
Remoting question

### DISCUSSION THREAD

**Discussion by StrongGesmbH** [quartznet/quartznet#777] [General] [answered]
From the last Migration Guide.
What does 
Remoting is currently only supported when running on full framework version.

mean ?

I was trying to implement this into a asp net core console app and it is not possible.
Is there any plan for supporting Remoting in asp net core ?

**Comment by Sicos1977** [comment]
.NET remoting is not available in .NET core... so I guess not.

**Comment by StrongGesmbH** [comment]
Remoting will be implemented in net core 3

**Comment by recepaslan** [comment]
> Remoting will be implemented in net core 3

Looks like they drop the remoting all together. https://docs.microsoft.com/en-us/dotnet/core/porting/net-framework-tech-unavailable#remoting

**Comment by lahma** [comment, marked as answer]
You probably want to keep an eye on https://github.com/quartznet/quartznet/issues/511 , this would require a HTTP based API to work. Closing this as question / duplicate.
